src/mono/wasm/runtime/snapshot.ts

```diff
@@ -168,6 +168,7 @@ async function getCacheKey(): Promise<string | null> {
     delete inputs.diagnosticTracing;
     delete inputs.appendElementOnExit;
     delete inputs.assertAfterExit;
+    delete inputs.interopCleanupOnExit;
```

Comment:
I added the comment, but maybe we need to revert the way how we calculate the hash. Instead of excluding the fields, we should include them. Or even assert that only expected fields left in it.